docker/dev/docker-compose.yml

```diff
@@ -0,0 +1,73 @@
+version: "3.6"
+services:
+  es_nginx:
+    image: es_dev_nginx
+    build:
+      context: ./../..
+      dockerfile: docker/nginx/Dockerfile
+      target: es_nginx
+    volumes:
+      - ./../..:/var/www
+    ports:
+      - 5000:80
+    networks:
+      - internal
+    depends_on:
+      - es_php_fpm
+  es_php_fpm:
+    image: es_dev_php_fpm
+    build:
+      context: ./../..
+      dockerfile: docker/dev/Dockerfile
+      target: es_php_fpm
+    user: "${UID}:${GID}"
+    volumes:
+      - ./../..:/var/www
+    environment:
+      MYSQL_HOST: es_database
+      MYSQL_USER: engelsystem
+      MYSQL_PASSWORD: engelsystem
+      MYSQL_DATABASE: engelsystem
+      PHP_IDE_CONFIG: serverName=engelsystem
+    networks:
+      - internal
+      - database
+    depends_on:
+      - es_database
+  es_workspace:
+    image: es_dev_workspace
+    build:
+      context: ./../..
+      dockerfile: docker/dev/Dockerfile
+      target: es_workspace
+    user: "${UID}:${GID}"
+    volumes:
+      - ./../..:/var/www
+    environment:
+      MYSQL_HOST: es_database
+      MYSQL_USER: engelsystem
+      MYSQL_PASSWORD: engelsystem
+      MYSQL_DATABASE: engelsystem
```

Comment:
We should also add
```yml
HOME: /tmp
```

To prevent yarn from showing unnecessary warnings like
```
Error: EACCES: permission denied, open '/.yarnrc
```
```
warning Skipping preferred cache folder "/.cache/yarn" because it is not writable.
[...]
warning Cannot find a suitable global folder. Tried these: "/usr/local, /.yarn"
[...]
┌──────────────────────────────────────────────┐
│           npm update check failed            │
│     Try running with sudo or get access      │
│     to the local update config store via     │
│ sudo chown -R $USER:$(id -gn $USER) /.config │
└──────────────────────────────────────────────┘
```
and so on

Reply:
✓ done